src/renderer/components/NodeSelectorPanel/RegularAccordionItem.tsx

```diff
@@ -25,45 +27,67 @@ const RegularAccordionItem = memo(
     ({ subcategoryMap, category, collapsed }: RegularAccordionItemProps) => {
         return (
             <AccordionItem key={category}>
-                <AccordionButton>
-                    <HStack
-                        flex="1"
-                        h={6}
-                        textAlign="left"
-                        verticalAlign="center"
-                    >
-                        <Center>
-                            <IconFactory
-                                accentColor={getNodeAccentColor(category)}
-                                icon={category}
-                            />
-                        </Center>
-                        {!collapsed && (
-                            <Heading
-                                size="5xl"
-                                textOverflow="clip"
-                                whiteSpace="nowrap"
-                            >
-                                {category}
-                            </Heading>
-                        )}
-                    </HStack>
-                    <AccordionIcon />
-                </AccordionButton>
+                <Tooltip
+                    closeOnMouseDown
+                    hasArrow
+                    borderRadius={8}
+                    fontSize="1.05rem"
+                    isDisabled={!collapsed}
+                    label={<ReactMarkdown>{`**${category}**`}</ReactMarkdown>}
```

Comment:
Could you please not use a whole Markdown renderer just to do `<strong>{category}</strong>`?

Same for the other non-Markdown usages as well.

Reply:
The point is that you don't need md to make text bold.